app/components/preprint-status-banner/component.js

```diff
@@ -0,0 +1,99 @@
+import Ember from 'ember';
+
+const PENDING = 'pending';
+const ACCEPTED = 'accepted';
+const REJECTED = 'rejected';
+
+const PRE_MODERATION = 'pre-moderation';
+const POST_MODERATION = 'post-moderation';
+
+export const ICONS = Object.freeze(Ember.Object.create({
+    [PENDING]: 'fa-hourglass-o',
+    [ACCEPTED]: 'fa-check-circle-o',
+    [REJECTED]: 'fa-times-circle-o'
+}));
+
+export const STATUS = Object.freeze(Ember.Object.create({
+    [PENDING]: 'components.preprint-status-banner.pending',
+    [ACCEPTED]: 'components.preprint-status-banner.accepted',
+    [REJECTED]: 'components.preprint-status-banner.rejected'
+}));
+
+export const MESSAGE = Object.freeze(Ember.Object.create({
+    [PRE_MODERATION]: 'components.preprint-status-banner.message.pending_pre',
+    [POST_MODERATION]: 'components.preprint-status-banner.message.pending_post',
+    [ACCEPTED]: 'components.preprint-status-banner.message.accepted',
+    [REJECTED]: 'components.preprint-status-banner.message.rejected'
+}));
+
+export const WORKFLOW = Object.freeze(Ember.Object.create({
+    [PRE_MODERATION]: 'components.preprint-status-banner.pre_moderation',
+    [POST_MODERATION]: 'components.preprint-status-banner.post_moderation'
+}));
+
+export const CLASS_NAMES = Object.freeze(Ember.Object.create({
+    [PRE_MODERATION]: 'preprint-status-pending-pre',
+    [POST_MODERATION]: 'preprint-status-pending-post',
+    [ACCEPTED]: 'preprint-status-accepted:',
+    [REJECTED]: 'preprint-status-rejected'
+}));
+
+
+export default Ember.Component.extend({
+    i18n: Ember.inject.service(),
```

Comment:
Nope! I assumed it was but it's only needed for translation in the javascript file.